src/main/frontend/handler/repeated.cljs

```diff
@@ -33,6 +33,7 @@
     "+"
     "Dotted"
     ".+"
+    "DoublePlus"
```

Comment:
"++" here will be the default/last case if both "Plus" and "Dotted" are not matched.

Reply:
Right. But other places seem to expect "DoublePlus", not "++"...